src/Action/BulkAction.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Crud\Action;
+
+use Cake\Network\Exception\NotImplementedException;
+use Cake\Event\Event;
+use Cake\ORM\Query;
+use Crud\Event\Subject;
+use Crud\Error\Exception\ActionNotConfiguredException;
+
+/**
+ * Handles 'Bulk' Crud actions
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ */
+abstract class BulkAction extends BaseAction
+{
+    /**
+     * Default settings for 'lookup' actions
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'enabled' => false,
+        'scope' => 'table',
+        'findMethod' => 'all'
+    ];
+
+    /**
+     * Handle a bulk event
+     *
+     * @return void
+     */
+    protected function _handle()
+    {
+        if (!method_exists($this, '_bulk')) {
+            throw new NotImplementedException(sprintf(
+                'Action %s does not implement a _bulk handler',
+                get_class($this)
+            ));
```

Comment:
It would be better to just make `abstract function _bulk` instead of dynamically checking on each request.